ati_netcanoem_ft_driver/src/ati_netcanoem_ft_driver_node_main.ros2.cpp

```diff
@@ -0,0 +1,16 @@
+#include <memory>
+
+#include <rclcpp/rclcpp.hpp>
+
+#include "ati_netcanoem_ft_driver_node.ros2.hpp"
+
+int main(int argc, char * argv[])
```

Comment:
Can you explain why this separate header + main file is necessary? The Robotiq F/T driver doesn't have it.

Reply:
@calderpg-tri that's actually an omission in the other driver. 

We're including this source file explicitly (instead of letting [`rclcpp_components_register_node`](https://github.com/ros2/rclcpp/blob/d3c0049b24f91b9bacdfa255dbdee8297862118c/rclcpp_components/cmake/rclcpp_components_register_node.cmake#L61-L72) implicitly generate it) so that Bazel consumers don't have to. The header file exposes the `AtiNetCanOemDriverNode` component so that we can statically link it (instead of relying on `dlopen` invocations and environments variables or RPATHs).